vignettes/articles/bp_literature.Rmd

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
I was planning to render it as an article with pkgdown and usethis::use_article() places it in the artcles folder under vignettes. I didn't manually place it there. Moreover, I've removed it now as it was basically a bibliography list. I figured it could be better placed elsewhere.